Watchman.Discord/Areas/Protection/Models/ServerSafeUsers.cs

```diff
@@ -1,59 +1,71 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Linq;
+using Autofac;
+using Devscord.DiscordFramework.Middlewares.Contexts;
 using Devscord.DiscordFramework.Services;
 using Watchman.DomainModel.Messages;
 
 namespace Watchman.Discord.Areas.Protection.Models
 {
-    public struct ServerSafeUsers
+    public readonly struct ServerSafeUsers
     {
-        public static UsersService UsersService { get; set; }
+        public static IComponentContext ComponentContext { get; set; }
```

Comment:
lepiej nie przesadzać z tym ComponentContext

Reply:
chciałem tak aby nie przypisywać statycznie tylu serwisów w klasie wyżej

Reply:
zawsze możesz zrobić to jako po prostu pojemnik na dane
a nie rozbudowaną logike 

albo przekazywać przez parametry metod referencje do klas

Reply:
lepiej wrócić do poprzedniego sposobu, czyli statyczne pola z UsersService i DiscordServersService, czy do ctora przekazywać referencję do tych klas za każdym razem jak tworzę obiekt?

Reply:
myśle że można po prostu przekazywać referencje